.github/pull_request_template.md

```diff
@@ -33,3 +33,19 @@ Fixes: #issue_number
 
 List items that are not part of the PR and do not impact it's
 functionality, but are work items that can be taken up subsequently.
+
+---
+
+<details>
+<summary>Show available bot commands</summary>
+
+These commands are normally not required, but in case of issues, leave any of
+the following bot commands in an otherwise empty comment in this PR:
+
+- `@mergifyio rebase`: rebase on current HEAD of the target branch
```

Comment:
This command needs additional permissions I guess, I tried to run this on the other day and got "pkalever not authorized to use the command" or something similar

Reply:
here: https://github.com/ceph/ceph-csi/pull/1560#issuecomment-713350865

Reply:
Oh, really? You should be able to do it on your own PR, I think?

Reply:
ok, that seems to be only available for maintainers, and there is no configuration option (that I can find) to allow others